slsReceiverSoftware/include/BinaryFileStatic.h

```diff
@@ -40,36 +41,33 @@ class BinaryFileStatic {
 	 * @param unitindex unit index
```

Comment:
I would correct the comment to say it is the sub frame index and not frame number index. I would also remove frindexenable from the comment. I would use "subfIndex" rather than fnum as a variable name.

Reply:
ok, done